### PR TITLE
Transcript for call 14  #25

### PR DESCRIPTION
**Pull request by roleengineer**
Ready for review

### CODE REVIEW

**Comment by oskarth** [comment]
@mandrigin @PombeirP please have a sanity check, one more approve then we can merge

**Comment by oskarth** [comment]
Fixed some bad links in README btw (missing i in a few links), please rebase

File: notes/meeting14.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+### DApp Store
+- Moving into implementation of Maciej's DApp store designs (web DApp)
+- Using an agency in Cape Town
+- Version 1 will be a categorized list
+- In the meantime, contract will be prepared for audit
+
+### Teller Network
+- Finishing the escrow work flow and UI details
+- Created a ___ system which makes it easier to see and test the DApp
```
Comment (PombeirP) [inline notes/meeting14.md]:
```suggestion
- Created a data seeding system which makes it easier to see and test the DApp
```

File: notes/meeting14.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+### Embark
+- Fixing bugs, planning to release beta this week and final 4 version next week
+- Launching new website next week as well
+
+Ricardo - liquid democracy, token balances, Whisper J (version 6 from version 3) - should make it easy to build something for Status in Ethereum J
+
+Anna - released hot fix last week mainly to "cancel" a pinned mail server (and opt back into automated selection). Also added changes to the bottom navigation in mobile - always visible now.
+
+Pedro - we're moving from our ad hoc build system where we download all dependencies we need into next package manager system. This has been on for 3-4 weeks now, we're in final stages before merging to develop. Testing soon. Last remaining items are fixes for Linux and Ubuntu 18. No impact on your existing environment, please to test. External developers are finding it easier to build.
```
Comment (PombeirP) [inline notes/meeting14.md]:
```suggestion
Pedro - we're moving from our ad-hoc build system where we download all dependencies we need to the Nix package manager system. This has been ongoing for 3-4 weeks now, we're in final stages before merging to develop. Testing soon. Last remaining items are fixes for Linux and Ubuntu 18. No impact on your existing environment, please test. External developers are finding it easier to build.
```

**Comment by oskarth** [comment]
Thanks @roleengineer, please address merge conflicts and we can merge this one

**Comment by oskarth** [comment]
Fixed

**Pull request merged**